README.md

```diff
@@ -47,17 +50,25 @@ Default values are filled in [config.default.sh](scripts/config.default.sh).
 - Create *config.sh* with this content:
   - ```
     #!/bin/bash
+    # Define required values.
     root_dir="<your topological-inventory root dir>"
+    MY_GITHUB_NAME="<your github name>"
+    MY_GITHUB_TOKEN="<your github token>"
+    ACCOUNT_NUMBER="<your account number>"
+    # Define optional values, as needed.
+    RVM_RUBY_VERSION_TP_INV="<your ruby version>"
```

Comment:
Great idea to add these @roliveri. Should it be noted that you need to include the patch too? e.g. is it `2.7pxxx` or `ruby2.7pxxx` or `ruby2.7`. I had that question when I ran the scripts.

Reply:
I set it to `2.5.3`. But I don't know all `rvm use` formats

Reply:
My intent wasn't to actually document the format of the Ruby version, but only to document that the version can be optionally overridden. My view is, when a user chooses to override the version, they should be aware of the version they want (including patch number if applicable).

Reply:
@roliveri can you add a comment "in 'rvm use' format"? Then it'll be clear, I think

Reply:
@slemrmartin Done.

I've also Updated repositories.sh to conditionally set `start_by_default`, so it can be more easily overridden in config.sh.

Reply:
Great!